gradle.properties

```diff
@@ -1,6 +1,17 @@
+# Disable buildFeatures flags by default
+android.defaults.buildfeatures.aidl=false
+android.defaults.buildfeatures.buildconfig=false
+android.defaults.buildfeatures.renderscript=false
+android.defaults.buildfeatures.resvalues=false
+android.defaults.buildfeatures.shaders=false
+
 android.useAndroidX=true
 android.enableJetifier=false
 android.nonTransitiveRClass=true
-org.gradle.jvmargs=-Xmx1400m
```

Comment:
This value was intentionally low so people with hardware that is not very powerful can still contribute. If you have a suggestion on how to use a low default value, but have an easy way to override it for CI, I'm all ears.

Reply:
31ddac19c312266521636e1f4f7a8dc1f0cf01f7